BasicOrderManagementProcess/BasicOrderManagementProcess_proStep.md

```diff
@@ -0,0 +1,157 @@
+
+> [Info] The tax rate to be applied depends on the ship-to country, the tax zone and the product type. Based on these parameters, a decision matrix determines automatically the tax rate to be applied in the invoice. This decision matrix is configured in the *Taxes* module, see [Manage the decision matrix](#to-be-determined).
+
+
+### 5. Payment handling  
+
+![Payment handling](../Assets/Screenshots/BasicOrderManagementProcess/OrderManagementProcess_5.jpg "[Payment handling]")
+
+After receiving the authorization from the sales channel, the *Payments* module sends a request to the PSP to capture the payment. If the payment capture is successful, the PSP sends a confirmation back to the *Payments* module. The payment process is considered as paid by the *Actindo Core1 Platform*.
+
+The payment is then sent to the *Accounting* module via the *Payment processing* module and matched with the open item based on different relevant data, such as amount, customer number, invoice number or payment reference. If the matching is successful, the open item is cleared. The order is considered as fully paid and its payment status accordingly updated in the *Order Management* module. 
+
+> [Info] The *Payment processing* module serves as link between any available payment source and the *Accounting* module. Other payment methods, such as bank transfer or direct debit, can also be configured, as long as the necessary process steps are defined in a workflow. 
+
+
+### 6. From the warehouse to the customer via fulfiller  
+
+![From warehouse to customer via fulfiller](../Assets/Screenshots/BasicOrderManagementProcess/OrderManagementProcess_6.jpg "[From warehouse to customer via fulfiller]")
+
+Companies usually outsource the order fulfillment to a third-party logistics provider. In this case, the *Fulfillment* module acts as link between the *Actindo Core1 Platform* and the fulfillment partner's external system. Both systems are connected and communicate via driver.
+
+On payment confirmation and based on the delivery note, a dispatch note is generated in the *Fulfillment* module. A dispatch note is a request for the fulfiller to pick, pack and send the ordered material to the customer. The dispatch note contains, at least, the ordered line items, including quantity and SKU, that must be sent to the customer.
+
+The fulfiller confirms receival of the dispatch note and begins processing the request. When all ordered line items have been sent to the customer, the fulfiller confirms the goods dispatch with one or, if necessary, several shipments. The dispatch note and the shipment status are updated accordingly in the *Fulfillment* module via driver.
+
+> [Info] Some companies may also have an internally managed warehouse. In this case, the *Fulfillment* module is not necessary and this process step is omitted altogether.  
+
+[comment]: <> (Julian: hier fehlte der Prozess in Janniks Notizen. Wollen wir ausführlich hier was  dazu erklären oder ist es unnötig?)
+
```

Comment:
Info to be added:
If your company has an internally managed warehouse... 
The Order Management module allows you to create picklists based on the delivery notes. Actindo IQS (Intelligent Quick Shipment) module available to help by the internal warehouse management, e.g. pick and pack (printout of labels, packing, etc.)